src/include/buffer.h

```diff
@@ -1177,6 +1173,9 @@ namespace buffer CEPH_BUFFER_API {
     void append_zero(unsigned len);
     void prepend_zero(unsigned len);
 
+    reserve_t obtain_contiguous_space(unsigned len);
+    reserve_t obtain_contiguous_space_rounded(unsigned len);
```

Comment:
where is this function used?

Reply:
I think is was a leakage from [`contigous_reserver`](https://gist.github.com/rzarzynski/f9515eb06c170c7c9658e9824feccd8f).